osu.Game/Overlays/Profile/Header/BadgeContainer.cs

```diff
@@ -0,0 +1,190 @@
+﻿// Copyright (c) 2007-2018 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using System;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Cursor;
+using osu.Framework.Graphics.Shapes;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Graphics.Textures;
+using osu.Framework.Input;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Users;
+using OpenTK;
+
+namespace osu.Game.Overlays.Profile.Header
+{
+    public class BadgeContainer : Container
+    {
+        private const float outer_container_width = 98;
+        private const float outer_container_padding = 3;
+        private static readonly Vector2 badge_size = new Vector2(outer_container_width - outer_container_padding * 2, 46);
```

Comment:
I'd really like this to be reversed. `badge_size` should be 86x40, and everything else should be padded around that to achieve the desired sizes.

As it currently is, I had to go into draw vis + check childsize to make sure that the badges were the correct size.